lib/python/treadmill_aws/hostmanager.py

```diff
@@ -43,10 +59,15 @@ def create_host(ec2_conn, ipa_client, image_id, count, domain,
     if instance_vars is None:
         instance_vars = {}
 
+    if hostname and r'-{time}' not in hostname and count > 1:
```

Comment:
just maintain set of already created hostnames. if new hostname in the set, raise. i do not want to hardcode -{time} as only pattern.